components/namespace-lister/base/service.yaml

```diff
@@ -3,11 +3,13 @@ kind: Service
 metadata:
   name: namespace-lister
   namespace: namespace-lister
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: namespace-lister-tls
```

Comment:
I would expect to see a change to namespace-lister deployment as well for mounting the tls certificates.

Reply:
namespace-lister is not supporting TLS at the moment. let's move this to a future PR.

Reply:
I've filed https://github.com/konflux-ci/namespace-lister/pull/29 to enable it in namespace-lister.